src/renderer/components/primitives/Header.tsx

```diff
@@ -0,0 +1,51 @@
+import { type FC, useContext } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { ArrowLeftIcon, type Icon } from '@primer/octicons-react';
+
+import { Heading, IconButton, Octicon, Stack } from '@primer/react';
+import { AppContext } from '../../context/App';
+import { Size } from '../../types';
+
+interface IHeader {
+  icon: Icon;
+  children: string;
+  fetchOnBack?: boolean;
+}
+
+export const Header: FC<IHeader> = (props: IHeader) => {
+  const navigate = useNavigate();
+
+  const { fetchNotifications } = useContext(AppContext);
+
+  return (
+    <div className="mx-6 mt-2 py-2">
+      <Stack direction="horizontal" justify="space-between">
+        <IconButton
+          aria-label="Go Back"
```

Comment:
```suggestion
          aria-labelledby="Go Back"
```
Personally, I'd remove the tooltip from the back button by using `aria-labelledby` instead 👀

Reply:
Good callout.  Switching to `aria-labelledby`